src/Reader/StandaloneExtensionManager.php

```diff
@@ -65,14 +65,9 @@ public function add($name, $class)
      * Remove an extension.
      *
      * @param string $name
-     * @return bool
      */
     public function remove($name)
     {
-        if (array_key_exists($name, $this->extensions)) {
```

Comment:
Any reason, why this check is removed?

Reply:
it is not needed. unset will work the same and there is no distinction between removing instance and noop.

Reply:
@Xerkus 
You are right, it throws no error.

Reply:
It's because `unset` is a language construct, not a real function, and it doesn't care whether the thing you're unsetting exists or not. The check was only there before to set the return value.